kolibri/plugins/learn/assets/src/views/CardList.vue

```diff
@@ -263,10 +262,6 @@
     text-decoration: none;
   }
 
-  .mobile-card {
```

Comment:
[Nice to have] Since you removed `.mobile-card` class styles, it'd be good if you could clean up its assignments from the component markup, for example

```
:class="{ 'mobile-card': isMobile }"
```
as this code is now obsolete.